nservicebus/messaging/messages-events-commands.md

```diff
@@ -20,72 +20,65 @@ redirects:
+_Can_ be sent using the [gateway](/nservicebus/gateway). | _Cannot_ be sent using the [gateway](/nservicebus/gateway).
 
-### Command
+Note: In a request and response pattern, _reply_ messages are neither a command nor an event.
 
-Used to request that an action should be taken. A *Command* is intended to be _sent to a receiver_ (all commands should have one logical owner and should be sent to the endpoint responsible for processing). As such, commands:
 
- * Are not allowed to be _published_.
- * Cannot be _subscribed_ to or _unsubscribed_ from.
- * Cannot implement `IEvent`.
+### Validation
 
+There are checks in place to ensure best practices are followed. Violations of the above guidelines generate the following exceptions:
 
-### Event
-
-Used to communicate that some action has taken place. An *Event* should be _published_. An event:
-
- * Can be _subscribed_ to and _unsubscribed_ from.
- * Cannot be sent using `Send()` (since _all events should be published_).
- * Cannot implement `ICommand`.
- * Cannot be sent using the gateway, i.e. `SendToSites()`.
+partial: errors
 
-Note: For reply messages in a request and response pattern, use `IMessage` since these replies are neither a Command nor an Event.
 
+ ## Designing messages
 
-### Validation Messages
+ Messages represent data contracts between endpoints. They should be designed according to the following guidelines.
+ 
+ Messages should:
 
-There are checks in place to ensure following of the best practices. While violating above rules the following exceptions can be seen:
+ * be simple [POCO](https://en.wikipedia.org/wiki/Plain_old_CLR_object) objects.
+ * be as small as possible.
+ * satisfy the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). Types used for other purposes (e.g. domain objects, data access objects, or UI binding objects) should not be used as messages.
 
-partial: errors
 
+## Defining messages
 
-## Defining Messages
-
-Messages can be defined via *interfaces* or via *conventions*.
+Messages can be defined either by *interfaces* or *conventions*.
 
 
 ### Interfaces
 
-The simplest way to define a message is to use interfaces.
+The simplest way to define messages is to use interfaces.
 
- * `NServiceBus.IMessage` for defining a Message.
- * `NServiceBus.ICommand` for defining a Command.
- * `NServiceBus.IEvent` for defining an Event.
+ * `NServiceBus.ICommand` for a command.
+ * `NServiceBus.IEvent` for an event.
+ * `NServiceBus.IMessage` for any other type of message (e.g. a _reply_ in a request response pattern).
 
 ```cs
-public class MyMessage : IMessage { }
-
 public class MyCommand : ICommand { }
 
 public class MyEvent : IEvent { }
 
 public interface MyEvent : IEvent { }
+
+public class MyMessage : IMessage { }
 ```
 
+NOTE: This approach has some drawbacks, as described in ["Unobtrusive mode messages"](unobtrusive-mode.md).
```

Comment:
We don't normally use quotes when referencing other articles. Is this a new convention?

Reply:
👍 I'll remove them.